highfive/configs/rust-lang/rust.json

```diff
@@ -201,7 +196,7 @@
             "reviewers": ["@rust-lang/miri"]
         },
         "library": {
-            "message": "Hey! It looks like you've submitted a new PR for the library teams!\n\nIf this PR contains changes to any `rust-lang/rust` public library APIs then please comment with `r? rust-lang/libs-api @rustbot label +T-libs-api -T-libs` to request review from a libs-api team reviewer. If you're unsure where your change falls no worries, just leave it as is and the reviewer will take a look and make a decision to forward on if necessary.\n\nExamples of `T-libs-api` changes:\n\n* Stabilizing library features\n* Introducing insta-stable changes such as new implementations of existing stable traits on existing stable types\n* Introducing new or changing existing unstable library APIs (excluding permanently unstable features / features without a tracking issue)\n* Changing public documentation in ways that create new stability guarantees\n* Changing observable runtime behavior of library APIs",
+            "message": "Hey! It looks like you've submitted a new PR for the library teams!\n\nIf this PR contains changes to any `rust-lang/rust` public library APIs then please comment with `@rustbot label +T-libs-api -T-libs` to tag it appropriately. If this PR contains changes to any unstable APIs please include a link to the relevant [API Change Proposal](https://std-dev-guide.rust-lang.org/feature-lifecycle/api-change-proposals.html) or [create one](https://github.com/rust-lang/libs-team/issues/new?assignees=&labels=api-change-proposal%2C+T-libs-api&template=api-change-proposal.md&title=%28My+API+Change+Proposal%29) if you haven't already. If you're unsure where your change falls no worries, just leave it as is and the reviewer will take a look and make a decision to forward on if necessary.\n\nExamples of `T-libs-api` changes:\n\n* Stabilizing library features\n* Introducing insta-stable changes such as new implementations of existing stable traits on existing stable types\n* Introducing new or changing existing unstable library APIs (excluding permanently unstable features / features without a tracking issue)\n* Changing public documentation in ways that create new stability guarantees\n* Changing observable runtime behavior of library APIs",
```

Comment:
We should probably say something like "edit PR description" or "leave a comment" rather than include; include feels confusing to me.

Reply:
updated it to suggest editing the PR description, seems easier to navigate that way vs leaving comments.